src/elements/a-gltf-entity.js

```diff
@@ -284,7 +287,6 @@ AFRAME.registerElement("a-gltf-entity", {
         if (attr === "src") {
           this.setSrc(newVal);
         }
-        AFRAME.AEntity.prototype.attributeChangedCallback.call(this, attr, oldVal, newVal);
```

Comment:
intentional? Not well versed in this code but this seems like a impactful change.

Reply:
Hmm, this is calling the AEntity version of attributeChangedCallback though, doesnt this still need to happen so that other components on the `a-gltf-entity` get their values updated?